x-pack/plugins/maps/public/classes/util/can_skip_fetch.ts

```diff
@@ -93,63 +101,63 @@ export async function canSkipSourceUpdate({
   let updateDueToTime = false;
   let updateDueToTimeslice = false;
   if (timeAware) {
-    updateDueToApplyGlobalTime = prevMeta.applyGlobalTime !== nextMeta.applyGlobalTime;
-    if (nextMeta.applyGlobalTime) {
-      updateDueToTime = !_.isEqual(prevMeta.timeFilters, nextMeta.timeFilters);
-      if (!_.isEqual(prevMeta.timeslice, nextMeta.timeslice)) {
-        updateDueToTimeslice = getUpdateDueToTimeslice(nextMeta.timeslice);
+    updateDueToApplyGlobalTime = prevMeta.applyGlobalTime !== nextRequestMeta.applyGlobalTime;
+    if (nextRequestMeta.applyGlobalTime) {
+      updateDueToTime = !_.isEqual(prevMeta.timeFilters, nextRequestMeta.timeFilters);
+      if (!_.isEqual(prevMeta.timeslice, nextRequestMeta.timeslice)) {
+        updateDueToTimeslice = getUpdateDueToTimeslice(nextRequestMeta.timeslice);
       }
     }
   }
 
   let updateDueToRefreshTimer = false;
-  if (refreshTimerAware && nextMeta.refreshTimerLastTriggeredAt) {
+  if (refreshTimerAware && nextRequestMeta.refreshTimerLastTriggeredAt) {
```

Comment:
Is `refreshTimerAware` and `refreshTimerLastTriggeredAt` check even needed anymore? Won't this case be captured in mustForceRefresh?

Reply:
thx. this is a good catch. `refreshTimerLastTriggeredAt` is never updated, so never used. which also means that `IVectorSource#isRefreshTimerAware()` is no longer necessary either. 

Removed both. A nice cleanup of the interfaces.